packages/amplify-e2e-core/src/categories/geo.ts

```diff
@@ -87,6 +99,73 @@ export function addPlaceIndexWithDefault(cwd: string, settings: GeoConfig = {}):
   });
 }
 
+/**
+ * Add geofence collection with default values. Assume auth and cognito group are configured
+ * @param cwd command directory
+ */
+ export function addGeofenceCollectionWithDefault(cwd: string, groupName: string, settings: GeoConfig = {}): Promise<void> {
+  const config = { ...defaultGeoConfig, ...settings };
+  return new Promise((resolve, reject) => {
+    const chain = spawn(getCLIPath(), ['geo', 'add'], { cwd, stripColors: true })
+      .wait('Select which capability you want to add:')
+      .sendKeyDown(2)
+      .sendCarriageReturn()
+      .wait('Provide a name for the Geofence Collection:')
+      .sendLine(config.resourceName)
+      .wait('Select one or more cognito groups to give access:')
+      .sendCarriageReturn()
+      .wait(`What kind of access do you want for ${groupName} users? Select ALL that apply:`)
+      .sendCtrlA()
+      .sendCarriageReturn()
+
+    if (config.isAdditional === true) {
+      chain.wait(defaultGeofenceCollectionQuestion);
+      if (config.isDefault === true) {
+        chain.sendConfirmYes();
```

Comment:
This any all other tests that interact with `amplify-prompts` should use `chain.sendYes()` and `chain.sendNo()` instead of `sendConfirmYes` and `sendConfirmNo`. sendConfirmYes/No sends an extra newline which is not necessary when using `amplify-prompts` and causes flaky tests because there's a race condition where the newline may get sent to the next prompt instead